src/library.js

```diff
@@ -1287,6 +1287,9 @@ LibraryManager.library = {
   llvm_eh_typeid_for: function(type) {
     return type;
   },
+#if DISABLE_EXCEPTION_CATCHING
+  __cxa_begin_catch: function() {}, // without exceptions, just do nothing
```

Comment:
I don't really understand why stub out this function, but none of the other ones (e.g. cxa_end_catch)?

Reply:
Mainly because I stubbed out what I saw appear in practice, but yeah, better to do something more general here. I pushed a hopefully nicer version - it stubs out the obvious functions, and in a cleaner way. 

Note we can't stub out things like `throw` since they still need to halt execution.